test/SignIn.test.js

```diff
@@ -0,0 +1,115 @@
+const app = require('../app');
```

Comment:
The file name shoul be `signIn.test.js`
First letter capitalized. THe same for all files.